qemu/tests/cfg/expose_host_mtu.cfg

```diff
@@ -0,0 +1,12 @@
+- expose_host_mtu: install setup image_copy unattended_install.cdrom
+    requires_root = yes
+    type = expose_host_mtu
+    kill_vm = yes
+    only virtio_net
+    start_vm = no
+    nic_extra_params = ",host_mtu=4000"
+    nm_stop_cmd = service NetworkManager stop
+    check_guest_mtu_cmd = "ifconfig %s"
+    bridge_get_if_cmd = "brctl show %s | awk '{print $4}' | tail -1"
```

Comment:
Please use `ip link` to replace `brctl` on rhel8

Reply:
Updated. Please help review again. Thanks.